libmogwai-schedule/tests/signal-logger.h

```diff
@@ -137,4 +137,84 @@ gchar           *mws_signal_logger_format_emissions (MwsSignalLogger
       } \
   } G_STMT_END
 
+/**
+ * mws_signal_logger_assert_notify_emission_pop:
+ * @self: a #MwsSignalLogger
+ * @obj: a #GObject instance to assert the emission matches
+ * @property_name: property name to assert the #GObject::notify signal matches
+ *
+ * Assert that a signal emission can be popped off the log (using
+ * mws_signal_logger_pop_emission()) and that it is an emission of
+ * #GObject::notify for @property_name on @obj. To examine the #GParamSpec for
+ * the notify emission, use mws_signal_logger_assert_emission_pop() instead.
+ *
+ * If a signal emission can’t be popped, or if it doesn’t match
+ * #GObject::notify, @property_name and @obj, an assertion fails, and some debug
+ * output is printed.
+ *
+ * Since: 0.1.0
+ */
+#define mws_signal_logger_assert_notify_emission_pop(self, obj, property_name) \
+  G_STMT_START { \
+    gpointer anep_obj = NULL; \
```

Comment:
It is not clear to me what does `anep` stands for. It is not an English word, apparently. Please consider using the full verbose name for it.

Reply:
It’s a contraction of `assert_notify_emission_pop`. I originally used the full phrase, but it made things *really* long. I think the variables in this macro (and the other macros) need a prefix to avoid their names being too generic, to avoid shadowing any variables in the calling scope.

Suggestions for better naming welcome; this is the best I could come up with. It (a contraction of the macro name) is what GLib uses in its GType/GObject macros.

Reply:
`pop_obj`? I agree `anep` reads weirdly.

Reply:
I’m worried that `pop_obj` might not be esoteric enough to avoid shadowing. Basically, we can only assert a claim on a namespace if it’s really related to the name of the macro. I’m going to go with `anep`.

Reply:
When writing macros, I usually prefix and suffix variables with an underscore. I don't know what's your personal preference there though.

As for the prefix, I suggest `notify_emission_pop_`. It's still long, but I think it's fine.

Notice that, from my side, this is not a blocker.